### ISSUE TITLE
start unmuted webvnc?

### ISSUE THREAD

**Issue by RezzZ** [jlesage/docker-baseimage-gui#137]
Is there a way to start an image based on this baseimage to _**not**_ start muted?

**Comment by jlesage** [comment]
Are you talking about audio from the app ? This is a browser limitation: a web page cannot play sound automatically without user interaction.

**Comment by RezzZ** [comment]
yes, audio when using the novnc/web interface to connect to the image. If the webbrowser always requires user interaction than there is nothing to be done here. Thank you for the reply!

**Comment by jlesage** [comment]
You are welcome!
